pom.xml

```diff
@@ -266,6 +272,21 @@
                 </plugin>
 
                 <plugin>
+                    <groupId>org.codehaus.gmavenplus</groupId>
+                    <artifactId>gmavenplus-plugin</artifactId>
+                    <version>${gmavenplus-plugin.version}</version>
+                    <dependencies>
+                        <dependency>
+                            <groupId>org.codehaus.groovy</groupId>
+                            <artifactId>groovy-all</artifactId>
+                            <version>${groovy.version}</version>
+                            <scope>runtime</scope>
+                        </dependency>
+                    </dependencies>
+                </plugin>
```

Comment:
In one of my iterations, I used some Groovy, but I ended up throwing the code out. But I thought it would be good to keep the option open in case we need it later.